llvm/lib/CodeGen/MIRPrinter.cpp

```diff
@@ -837,6 +837,8 @@ void MIPrinter::print(const MachineInstr &MI) {
     OS << "disjoint ";
   if (MI.getFlag(MachineInstr::NoUSWrap))
     OS << "nusw ";
+  if (MI.getFlag(MachineInstr::SameSign))
```

Comment:
Should get mir print / parse test

Reply:
Testing this via GISel.

Reply:
This test doesn't cover the parsing, see test/CodeGen/MIR/X86/fastmath.mir as an example

Reply:
Added, thanks, think it should be on track now.